tools/python/boutiques/publisher.py

```diff
@@ -15,19 +15,26 @@ class Publisher():
 
     def __init__(self, descriptor_file_name,
                  verbose, sandbox, no_int,
-                 auth_token):
+                 auth_token, replace, id):
```

Comment:
Should we give `replace` and `id` default values, to make their behaviour more clear? I understand that `replace=False` and `id=None` are the defaults. I realize this is only being called internally, so isn't *strictly* needed, it just increases the readability of code in modular pieces, in my opinion.

Reply:
Perhaps this also means doing the same for other params, `sandbox=False`, `verbose=False`, `no_int=False`, and bumping `auth_token` up in the list since it's required.

Reply:
Good idea. That way if we need to create a Publisher instance anywhere else in the code, we won't need to provide values for every single argument. The same should be done for other classes that have optional values.

Reply:
Exactly - we can create an issue go to back and do this for more, but want to just make that adjustment here for now?